### DISCUSSION TITLE
Add Delay to RPU file to synch video file

### DISCUSSION THREAD

**Discussion by Emilius256** [quietvoid/dovi_tool#282] [General]
Hi,
I wonder if it is possible to add a delay to the RPU file before injecting it into the video file to synchronize it.
I have a Video from two different sources, one is HDR10 and the other is Dolby Vision.
The first one it seems to me to have a better quality, and I would like to use geneate an RPU from the second video DV Layer and inject it to the first video.
Unfortunately, the second video have 24 black frames at the beginning.
It may be possible to add 24 black rames at the beginning of the RPU file or adding a Delay to have the frames in synch with the first video ?
Thanks for any help.

**Comment by hyllm** [comment]
just use editor... :) what about reading documentation? duplicate 24 frames from frame 0.

**Reply by Emilius256** [reply]
Thank you @hyllm, I thought it was much more complicated to use the editor.
